modules/nextflow/src/main/groovy/nextflow/cli/CmdRun.groovy

```diff
@@ -677,6 +682,45 @@ class CmdRun extends CmdBase implements HubOptions {
         return result
     }
 
+    /**
```

Comment:
Shouldn't be better to create an independent class in nf-commons, in case it is required for other commands or plugins in the future?